app.py

```diff
@@ -38,6 +38,26 @@ def mirror(name):
     }
     return create_response(data)
 
+@app.route('/users')
+def users():
+    data = {
+        'users': db.get('users')
+    }
+    return create_response(data)
+
+@app.route('/users/<id>')
+def user_by_id(id):
+    data = {
+        'user': db.getById('users', int(id))
+    }
+    if int(id) > 0 and int(id) < 5:
```

Comment:
you shouldn't hardcode this. if you add or delete users, this will no longer work. use what `db.getById` gives you to determine what to do